pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -88,10 +88,17 @@ var (
 	//location of the ecdh certificate
 	ecdhCertFile = types.IdentityDirname + "/ecdh.cert.pem"
 
+	//location of the attestation quote certificate
+	quoteCertFile = types.IdentityDirname + "/quote.cert.pem"
```

Comment:
I'm actually working in the background to reduce the amount of things that go into /conf partition. Can this NOT be managed in persist some place?

Reply:
Hi @rvs, thanks for reviewing this. I will change org-name to "The Linux Foundation",  Locality to "San Francisco", and also move the location of these quote certificates to /persist/config instead of /config.

Reply:
ECDH cert should be done the same as attest cert, both in terms or org and in terns of placement in filesystem.

Reply:
@eriknordmark Can you please take a look now? I've updated the changes.